pg_migration_tool/main.py

```diff
@@ -214,10 +214,10 @@ def generate_pg_dump_and_restore_cmd(self, event: Select.Changed)-> str:
     @on(Button.Pressed)
     def button_pressed(self, event: Button.Pressed):
         if event.button.id == "migrate":
-            event.button.disabled = True
-            self.query_one(Select).disabled = True
+            self.query_one(Log).clear()
```

Comment:
Will it not make it possible to accidentally click button multiple times and start multiple migrations at the same time?

Reply:
it will. what's your suggestion? is there a way to wait until process started by thread finishes?

Reply:
I would just disable button and restart the program after each migration. But I am ok with this as the chances of clicking twice are relatively small .

Reply:
then I'll go the safe way and disable them anyway and restart from scratch.

Reply:
@pawelros please review now